File: Solutions/3005- Count Elements With Maximum Frequency/Solution.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] arr = new int[101];
        int max = 0;
        for (int num : nums) {
            arr[num]++;
            if (arr[num] > max)
                max = arr[num];
        }

        int count = 0;
        for (int elem : arr) {
            if (elem == max)
                count += elem;
        }

        return count;
    }
}

File: Solutions/3005- Count Elements With Maximum Frequency/Solution.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        arr = [0] * 101
        max_num = 0

        for num in nums:
            arr[num] += 1
            if arr[num] > max_num:
                max_num = arr[num]

        count = 0
        for elem in arr:
            if elem == max_num:
                count += elem
        
        return count
        